packages/e2e-tests/puppeteer/scripts/rps.ts

```diff
@@ -1,6 +1,6 @@
 import {Page} from 'puppeteer';
 
-import {waitForAndClickButton, setUpBrowser, loadRPSApp} from '../helpers';
+import {waitForAndClickButton, setUpBrowser, loadWeb3App} from '../helpers';
```

Comment:
Not sure you want to touch this file either.

Reply:
See the comment above.